src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -1,25 +1,37 @@
---Made by Wilma456
-local args = {...}
+local Arg = {...}
```

Comment:
`args` is a standard name for a variable like this. It should be changed back to `args`